source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -26,31 +26,117 @@ content: |
 
      [MongoDBODBC]
      Description = ODBC for MongoDB BI Connector
-     Driver      = /usr/local/lib/<libmdbodbca.so|libmdbodbcw.so>
+     Driver      = /usr/local/lib/libmdbodbcw.so
      Trace       = Off
      TraceFile   = stderr
      ReadOnly    = yes
-     SERVER      = <server address>
-     PORT        = 3307
-     USER        = <username>
-     PASSWORD    = <password>
-     DATABASE    = <dbname>
+     Server      = <server address>
+     Port        = <port number>
+     User        = <username>
+     Password    = <password>
+     Database    = <dbname>
 
      ; optional SSL configuration
 
      SSLKEY      = </path/to/.pem-key-file>
      SSLCERT     = </path/to/ssl-certificate>
-     SSLMODE     = DISABLE|ALLOW|PREFER|REQUIRE
+     SSLMODE     = DISABLED|PREFERRED|REQUIRED
      SSLCA       = </path/to/ssl-ca-list-file>
      SSLCAPATH   = </path/to/ssl-ca-file-directory>
 
+  .. note::
+
+     All ``odbc.ini`` parameters are case-insensitive.
+
+  The following table describes the parameters used in the above DSN.
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 30 70
+
+     * - Parameter
+       - Description
+
+     * - Description
+       - Optional. Text description of the DSN.
+
+     * - Driver
+       - Path to ODBC driver file.
+
+     * - Trace
+       - ``On`` or ``Off``. Enable ODBC tracing.
+
+     * - Tracefile
+       - If ODBC tracing is enabled, the path to the file in which to
+         store the trace output.
+
+     * - ReadOnly
+       - ``yes`` or ``no``. Enforce a read-only ODBC session.
+
+     * - Server
+       - Address of the server where a |bi-short|
+         :binary:`~bin.mongosqld` instance is running.
+
+     * - Port
+       - Port number of the :binary:`~bin.mongosqld` instance.
+
+     * - User
+       - Username of the MongoDB user with permission to connect to
+         the :binary:`~bin.mongosqld` instance.
+
+     * - Password
+       - Password of the MongoDB user with permission to connect to
+         the :binary:`~bin.mongosqld` instance.
+
+     * - Database
+       - Name of the MongoDB database to use.
+
+     * - SSLKEY
+       - Path to an SSL key file to use for establishing a secure
+         connection. 
+
+     * - SSLCERT
+       - Name of the SSL certificate file to use for establishing a
+         secure connection. 
+
+     * - SSLMODE
+       - Allowable values are:
```

Comment:
I've moved to "Accepted values are:"

This makes it more clear that only these three values can be used. Allowable sounds like others could be used, but...

Reply:
There actually are a few other allowable values, but they're specialized for certain kinds of certificate verification and I'd rather not go there in this instance. There's a link to the MySQL reference page for users who need the complete list. Also you can use either "REQUIRE" or "REQUIRED", "DISABLE" or "DISABLED" etc.

Reply:
OK, changed it to "Accepted".